src/ntimporters/todoist/importer.py

```diff
@@ -74,7 +75,7 @@ def _import_project(project: dict):
                 author_id=models.Id16ReadOnly(id16()),
                 created_at=models.TimestampReadOnly(1),
                 last_event_at=models.TimestampReadOnly(1),
-                is_favorite=project.favorite,
+                is_favorite=project.is_favorite,
```

Comment:
i dont understand how it was working before

Reply:
https://developer.todoist.com/rest/v2/?python#migrating-from-v1 i know - they changed name.....